devdoc/interlocked_win32_requirements.md

```diff
@@ -0,0 +1,314 @@
+interlocked Win32
+================
+
+## Overview
+
+`interlocked Win32` is the Windows-specific implementation of the  `interlocked` module. Each function calls the corresponding `Interlocked` function from `windows.h`.
```

Comment:
>`Interlocked` function from `windows.h`. [](start = 127, length = 40)

again, you could consider linking to [https://docs.microsoft.com/en-us/windows/win32/sync/interlocked-variable-access](https://docs.microsoft.com/en-us/windows/win32/sync/interlocked-variable-access)

Not a big deal.